pitch.md

```diff
@@ -45,6 +60,7 @@ _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
+There are plenty of languages to solve constraint problems in general such as [AIMMS](https://en.wikipedia.org/wiki/AIMMS), [Oz](https://en.wikipedia.org/wiki/Oz_(programming_language)), and [Curry](https://en.wikipedia.org/wiki/Curry_(programming_language)) but none that are as specifically targeted as mine. They all are more GPL like for solving problems with constraints, but I think that my idea of a language that solves a particular constraint problem is unique.
```

Comment:
It might be interesting to see if somebody has implemented a group problem in something like Prolog, perhaps as an assignment. I think I've come across something like that once, I'll try to find it. Maybe similar would be an implementation of a scheduling program.